applications/web3-open-jurisdiction-stack.md

```diff
@@ -0,0 +1,346 @@
+
+- Will Sauer
+
+  - Will Sauer is a technically-grounded operations leader with a proven track record across the startup and venture ecosystem. Most recently he spent 3 years as a back-end software engineer. Prior to that he ran Finance and Operations at a Series A / B startup in San Francisco and worked for several alternative investment funds.
+
+### Team Code Repos
+
+- https://github.com/jurteam
+
+Please also provide the GitHub accounts of all team members. If they contain no activity, references to projects hosted elsewhere or live are also fine.
+
+- https://github.com/0xlyd
+- https://github.com/alessio
+- https://github.com/Polkaverse
+
+### Team LinkedIn Profiles (if available)
+
+- **Luca Yesupatham Daniel**: https://www.linkedin.com/in/0xlyd
+- **Alessio Treglia**: https://www.linkedin.com/in/alessiotreglia
+- **Pankaj Chaudhary**: https://www.linkedin.com/in/thepankajchaudhary
+- **Alessandro Palombo**: https://www.linkedin.com/in/alessandropalombo
+- **Raffaele Battaglini**: https://www.linkedin.com/in/raffaelebattaglini
+- **Luigi Cantisani**: https://www.linkedin.com/in/luigicantisani
+- **Will Sauer**: https://www.linkedin.com/in/will-sauer
+
+### Google Scholar Profiles (Or other research indexer profile, ex. Researchgate)
+
+- Alessandro Palombo ([link](https://scholar.google.com/citations?hl=en&user=YcIgha0AAAAJ&view_op=list_works&gmla=AJsN-F6ANDy4-SK-X7zCjxYItoNUDm9Dt70lST9gCqtWYLNSi33467lJlJGmznlhRt3gOnTwGLjs0pyQwTvXoEo0F4rkTnfnLXQin5VF1ebd_H2Wp_lHA1ezY8AxK3_Xy6m5FIXVYbPz))
+- Raffaele Battaglini ([link](https://scholar.google.com/citations?user=1wmRm_8AAAAJ&hl=it))
+
+## Development Status :open_book:
+
+In addition to our previous accomplishments, we have proactively initiated conversations with various stakeholders within the ecosystem, including representatives from Parity and Web3 Foundation as well as ecosystems' teams.
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Estimated Duration:** 4 months
+- **FTE:** 3 FTE
+- **Costs:** 80,000 USD
+
+The team involved has decided to work on a reduced market rate. If market rates were to be applied for professional or research activities, the cost would be prohibitively high. The team agrees to be paid on completion of each milestone.
+
+### Milestone 1 - Data Collection, Stakeholder Interviews and First Draft
+
+- **Estimated Duration:** 2 months
+- **FTE:** 3 FTE
+- **Costs:** 30,000 USDC
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a** | **Copyrights and Licenses** | Apache 2.0 |
+| **0b** | **Documentation**           | NA |
```

Comment:
What format/tech are you going to use to write the paper - latex? Or some kind of binary format?